pkg/devfile/detect.go

```diff
@@ -258,6 +258,9 @@ func (a AlizerClient) Analyze(path string) ([]model.Language, error) {
 // SelectDevFileFromTypes is a wrapper call to Alizer's SelectDevFileFromTypes()
 func (a AlizerClient) SelectDevFileFromTypes(path string, devFileTypes []model.DevFileType) (model.DevFileType, error) {
 	index, err := recognizer.SelectDevFileFromTypes(path, devFileTypes)
+	if err != nil {
+		return model.DevFileType{}, err
+	}
```

Comment:
Do we have test coverage for the case where a devfile is detected but there is an error? line 264

Reply:
Alizer only ever returns `-1` when there's an error

Reply:
ok, so you're saying the only time there's ever an error is when a -1 index is returned, otherwise it's nil - just checked the selectDevFileFromTypes implementation

Reply:
Yup. Your initial comment made me double check too 🙂